magenta/music/musicxml_parser_test.py

```diff
@@ -300,5 +313,40 @@ def testFluteScale(self):
     self.maxDiff = None
     self.assertProtoEquals(expected_ns, ns)
 
+  def testatonaltransposition(self):
+    """Test that transposition works when changing instrument transposition
```

Comment:
function docstrings should have a summary line that ends with ".", "?", or "!", followed by a blank line, and then followed by any additional information.

https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments